add citation and brief explanation of Matter (#28)
boucadair/encrypted-dns-forwarders: draft-rbw-home-servers.md

R-SUPPORT-CA: yes

R-SUPPORT-CLIENT:

>   : yes, if the clients add the home's Certification Authority to its trust list.

Reviewer: ```suggestion
  : yes, if the clients add the home's CA to their trust list.
```

R-REVOKE-AUTH:
  : Yes, user can update CRL on local certificate authority, and clients
    can retrieve the updated CRL.

## Matter

Home devices could be enrolled in {{Matter}} and client devices could
be configured to trust Matter certificates.

R-REDUCE-CA:
  : yes, it reduces interaction with public CAs but has same
number of interactions with the device vendor's CA to issue the
Matter Device Attestation Certificate (DAC) to each device.

R-ELIMINATE-CA: no

R-SUPPORT-CA: yes

R-SUPPORT-CLIENT:

>   : yes, if the clients add the Matter Product Attestation Intermediates (PAI) to its trust list.

Reviewer: ```suggestion
  : yes, if the clients add the Matter Product Attestation Intermediates (PAIs) to their trust list.
```
